examples/custom-logger/custom-logger.cpp

```diff
@@ -343,18 +343,14 @@ int main(int argc, char *argv[])
     logger->write();
 
     // To measure if your solution has actually converged, you can measure
-    // the error of the solution. one, neg_one are objects that represent
-    // the numbers which allow for a uniform interface when computing on any
-    // device. To compute the residual, all you need to do is call the apply
-    // method, which in this case is an spmv and equivalent to the LAPACK
-    // z_spmv routine. Finally, you compute the euclidean 2-norm with the
-    // compute_norm2 function.
-    auto one = gko::initialize<vec>({1.0}, exec);
-    auto neg_one = gko::initialize<vec>({-1.0}, exec);
-    auto res = gko::initialize<real_vec>({0.0}, exec);
-    A->apply(gko::lend(one), gko::lend(x), gko::lend(neg_one), gko::lend(b));
-    b->compute_norm2(gko::lend(res));
+    // the error of the solution. To compute the residual, all you need to do is
+    // call the apply method, which in this case is an spmv and equivalent to
+    // the LAPACK z_spmv routine. Finally, you compute the euclidean 2-norm with
+    // the compute_norm2 function.
+    auto res = gko::initialize<real_vec>({0.0}, exec->get_master());
+    A->apply(1.0, gko::lend(x), -1.0, gko::lend(b));
+    b->compute_norm2(lend(res));
 
     std::cout << "Residual norm sqrt(r^T r):\n";
-    write(std::cout, gko::lend(res));
+    write(std::cout, lend(res));
```

Comment:
this is confusing or not strong concern in ginkgo.
we use `gko::lend` not `lend` even if the compiler can find the correct lend from the input type.